clients/src/main/java/org/apache/kafka/clients/producer/internals/TransactionManager.java

```diff
@@ -432,7 +440,7 @@ void clearInFlightRequestCorrelationId() {
         inFlightRequestCorrelationId = NO_INFLIGHT_REQUEST_CORRELATION_ID;
     }
 
-    boolean hasInflightRequest() {
+    boolean hasInFlightRequest() {
```

Comment:
I think the idea is that `inflight` is one word (the config follows the same approach). Seems like that is an accepted spelling:

http://www.dictionary.com/browse/inflight